tests/code/test_transcripts.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from peewee import DoesNotExist
+
+from app.controllers.events.view_transcript import getSLCourseTranscript
+from app.controllers.events.view_transcript import getProgramTranscript
+from app.controllers.events.view_transcript import getUser
+from app.models.term import Term
+from app.models.course import Course
+from app.models.courseParticipant import CourseParticipant
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+
+@pytest.mark.integration
+def test_noGetSLTranscript():
+

```

Comment:
You're not actually testing your functions here. `User.get_by_id` is what is throwing the DoesNotExist error in all of these tests, not your functions.

Reply:
We moved `User.get_by_id` into our functions so that they don't accept users that are not in the database.